.gitignore

```diff
@@ -5,7 +5,12 @@
 .pydevproject
 .DS_Store
 .idea
-.vscode
+
+.vscode/*
+!.vscode/extensions.json
+!.vscode/settings.json
```

Comment:
Just remove from gitignore entirely? What else would live in this directory?

Reply:
I have a c_cpp_properties.json in there…must’ve been automatically created? I could delete it I suppose, but it’s annoying to have it constantly show as an untracked file.